addons/aws/1.0.1/kubeadm-cluster-config-v1beta2.yml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: ClusterConfiguration
+metadata:
+  name: kubeadm-cluster-configuration
+clusterName: $KUBERNETES_CLUSTER_NAME
+networking:
+  podSubnet: $POD_CIDR
```

Comment:
i dont know why this is here but my bet is it has something to do with `configure-cloud-routes`. you may want to similarly remove or make it optional.

Reply:
removing it. It feels unnecessary as the the podSubnet is created by Network plugins